common/src/Microsoft.Azure.IIoT.Hub.Processor/src/Services/EventProcessorFactory.cs

```diff
@@ -92,13 +101,14 @@ private class DefaultProcessor : IEventProcessor {
                 // Checkpoint if needed
                 if (_sw.ElapsedMilliseconds >= _interval) {
                     try {
-                        _logger.Debug("Checkpointing partition {partitionId}...", context.PartitionId);
+                        _logger.Debug("Checkpointing EventProcessor {id} for partition {partitionId}...",
+                            _processorId, context.PartitionId);
                         await context.CheckpointAsync();
                         _sw.Restart();
                     }
                     catch (Exception ex) {
-                        _logger.Debug(ex, "Failed checkpointing partition {partitionId}...",
-                            context.PartitionId);
+                        _logger.Debug(ex, "Failed checkpointing EventProcessor {id} for partition {partitionId}...",
```

Comment:
would it make sense to have this log as warning or information level?

Reply:
Yes, totally. I'll change it to `Warning()`.